src/Aspire.Hosting.GitHub.Models/GitHubModelsResource.cs

```diff
@@ -0,0 +1,43 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Aspire.Hosting.ApplicationModel;
+
+namespace Aspire.Hosting.GitHub.Models;
+
+/// <summary>
+/// Represents a GitHub Models resource.
+/// </summary>
+public class GitHubModelsResource : Resource, IResourceWithConnectionString
```

Comment:
Would this be better to be singular?

`GitHubModelResource`?

Having a plural in the name makes it sound plural, like it respresents more than one model.